projects/go-lib/src/lib/components/go-side-nav/go-nav-item/go-nav-item.component.scss

```diff
@@ -5,7 +5,6 @@
 .go-nav-item {
   align-items: center;
   display: flex;
-  padding-right: .5rem;
```

Comment:
Wasn't sure of the purpose of this and was making it impossible to align `launch` icon with dropdown arrow